### PR TITLE
Export query result (fixed sized values only) to arrow

### PR DESCRIPTION
**Pull request by ray6080**
- add `get_as_arrow()`, support BOOL, INT64, DOUBLE, DATE, TIMESTAMP, INTERVAL.
- rename `getResultValue` to `getValue`.

### CODE REVIEW

File: tools/python_api/test/test_arrow.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import sys
+
+sys.path.append('../build/')
+import kuzu
+import pyarrow as pa
+
+sys.path.append('../build/')
```
Comment (mewim) [inline tools/python_api/test/test_arrow.py:7]: Duplicated path

File: tools/python_api/requirements_dev.txt [hunk 2]
```diff
@@ -3,3 +3,4 @@ pytest
 pandas
 networkx~=3.0.0
 numpy
+pyarrow
```
Comment (mewim) [inline tools/python_api/requirements_dev.txt:6]: We should probably specify a version here in case the compatibility breaks.

**Review by mewim — APPROVED** [review]
Overall looks good to me. I am not very familiar with the internals of Arrow format though.

File: tools/python_api/test/test_arrow.py (hunk 1 above)
Comment (ray6080) [reply tools/python_api/test/test_arrow.py]: Done.

File: tools/python_api/requirements_dev.txt (hunk 2 above)
Comment (ray6080) [reply tools/python_api/requirements_dev.txt]: Done.

**Pull request merged**